tsconfig.json

```diff
@@ -0,0 +1,30 @@
+{
+  "compilerOptions": {
+    "outDir": "lib",
+    "module": "commonjs",
+    "target": "es6",
```

Comment:
Yeah that's what I meant :)
(And it will be fixed soon https://github.com/Microsoft/TypeScript/pull/12346)